log4j-1.2-api/src/test/resources/config-1.2/hadoop/hadoop-common-project/hadoop-kms/src/test/resources/log4j.properties

```diff
@@ -24,7 +24,7 @@ log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
 
 log4j.rootLogger=INFO, stdout
 log4j.logger.org.apache.hadoop.conf=ERROR
-log4j.logger.org.apache.hadoop.crytpo.key.kms.server=ALL
+log4j.logger.org.apache.hadoop.crypto.key.kms.server=ALL
```

Comment:
Google doesn't recognize this string outside here:
https://www.google.com/search?q=%22log4j.logger.org.apache.hadoop.crytpo.key.kms.server%22